pkg/konnector/controllers/cluster/serviceexport/cluster-scoped/utils.go

```diff
@@ -29,10 +29,13 @@ import (
 // a cluster-scoped object. Its value is the corresponding cluster namespace name.
 const ClusterNsAnnotationKey = "kube-bind.io/cluster-namespace"
 
+// Prepend adds clusterNs and a dash before name.
 func Prepend(name, clusterNs string) string {
 	return clusterNs + "-" + name
 }
 
+// Behead tries to remove clusterNs and a dash that goes before name.
+// If name doesn't start with clusterNs, name is returned unchanged.
```

Comment:
nit: sqaush this kind of commits into the commit where these functions were added. Makes life of the reviewer much easier.

Reply:
Done in latest push.